content/de/privacy.md

```diff
@@ -5,7 +5,7 @@
     "aliases": [ "privacy-policy", "data-protection", "privacy", "datenschutzerklaerung", "datenschutzerklärung", "privatsphaere", "privatsphäre" ]
 }
 
-Unsere Datenschutzerklärung wurde zuletzt am 13. September 2020 geändert.
+Unsere Datenschutzerklärung wurde zuletzt am 17. September 2021 geändert.
```

Comment:
Is this still correct? We'll have merged later.

Reply:
I have stopped caring about that since it usually takes a while for my PRs to be merged. If the merging person cares, they can update it.